objects/obj_ncombat/Alarm_0.gml

```diff
@@ -1,33 +1,31 @@
-
-
+// Sets up the number of enemies based on the threath level, enemy type and specific story events
 if (battle_special="cs_meeting_battle5") then alpha_strike=1;
 
 instance_activate_object(obj_enunit);
 
-
+// Checks if Chapter master is a psyker
```

Comment:
redundant, we already know this from `chapter_master_psyker=true`

a better comment would explain what `kamehameha` does